### PR TITLE
Docker Hub: publish image on push success

### LINKED ISSUE #4: Set up a CD pipeline for easier demo purposes
Today we don't have an online demo app, it has to be run on localhost. We need to procure a cloud account and set up a CD pipeline.

### PR DESCRIPTION
**Pull request by LoneRifle**
Adapt a recipe from Beeline to build the standalone image. Given the
time it takes to do so, and that our eventual setup will use one image
per _module_, we have to think about whether we wish to keep building
standalone images

Fixes #4

### CODE REVIEW

**Review by liangyuanruo — COMMENTED** [review]
lgtm so far

File: .travis.yml [hunk 1]
```diff
@@ -21,3 +21,47 @@ script:
   # check that git working tree is clean after running npm install via a frontend-maven-plugin
   # the git command returns 1 and fails the build if there are any uncommitted changes
   - git diff HEAD --exit-code
+after_success:
```
Comment (liangyuanruo) [inline .travis.yml:24]: to remove these echo / cat / ls statements

**Review by liangyuanruo — APPROVED** [review]
do consider wrapping each stage in a shell script for neatness.

**Comment by LoneRifle** [comment]
I'll clean up the script lines if/when we start distributing multiple images for deployment

**Pull request merged**